distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time_Formatter.enso

```diff
@@ -124,7 +124,7 @@ type Date_Time_Formatter
              Date.parse "1 Nov '95" "d MMM ''yy{2099}" == (Date.new 2095 11 01)
     @locale Locale.default_widget
     from_simple_pattern : Text -> Locale -> Date_Time_Formatter ! Date_Time_Format_Parse_Error
-    from_simple_pattern pattern:Text locale:Locale=Locale.default =
+    from_simple_pattern pattern:Text='' locale:Locale=Locale.default =
```

Comment:
minor thing: I see above in `Numbers.format` we have `""` and here `''`, shall we try to 'standardize' which one we prefer for empty strings? Or does this depend on the context?

How does the text input widget work? Does it always use `'`? Does it support escapes?

Reply:
I want to move to:
- Use single quotes by default.
- Use double quotes if it would avoid escaping.
- Use escaped single quotes.

No the text input doesn't do that it just uses escaped single quotes.